lib/util/visit-import.js

```diff
@@ -5,30 +5,37 @@
 "use strict"
 
 const path = require("path")
-const isCoreModule = require("is-core-module")
+const isBuiltinModule = require("is-builtin-module")
 const getResolvePaths = require("./get-resolve-paths")
 const getTryExtensions = require("./get-try-extensions")
 const ImportTarget = require("./import-target")
 const stripImportPathParams = require("./strip-import-path-params")
 
+/** @typedef {import('@typescript-eslint/typescript-estree').TSESTree.ImportDeclaration} ImportDeclaration */
+
+/**
+ * @typedef {Object} VisitImportOptions
```

Comment:
```suggestion
 * @typedef VisitImportOptions
```

When using `@property` no type is needed in `@typedef`, its implied that its an object